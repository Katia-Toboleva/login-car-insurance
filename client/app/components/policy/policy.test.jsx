import React from 'react';
import { shallow } from 'enzyme';
import Policy from './policy';

const getUnit = (props) => (
  shallow(<Policy {...props} />)
);

describe('Policy', () => {
  describe('render', () => {
    it('should render correctly', () => {
      const wrapper = getUnit({
          policyDetails: {
            policyRef: '123',
            cover: 'allIncl',
            car: 'bmw',
            address: '123 flat, city, postcode',
          },
        },
      );

      expect(wrapper).toMatchSnapshot();
    });
  });
});
