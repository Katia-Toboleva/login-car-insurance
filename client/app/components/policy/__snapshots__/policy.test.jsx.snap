// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Policy render should render correctly 1`] = `
<div
  className="policy__wrapper"
>
  <div
    className="policy"
  >
    <div
      className="policy__title"
    >
      <Text
        center={false}
        color="black"
        size="large"
        text="my policy"
        transform="capitalize"
        weight="bold"
      />
    </div>
    <div
      className="policy__body"
    >
      <Text
        center={false}
        color="black"
        size="medium"
        text="Policy reference:"
        weight="bold"
      />
      <Text
        center={false}
        color="black"
        size="medium"
        text="123"
      />
      <Text
        center={false}
        color="black"
        size="medium"
        text="Cover type:"
        weight="bold"
      />
      <Text
        center={false}
        color="black"
        size="medium"
        text="allIncl"
      />
      <Text
        center={false}
        color="black"
        size="medium"
        text="Car:"
        weight="bold"
      />
      <Text
        center={false}
        color="black"
        size="medium"
        text="bmw"
        transform="capitalize"
      />
      <Text
        center={false}
        color="black"
        size="medium"
        text="Address:"
        weight="bold"
      />
      <Text
        center={false}
        color="black"
        size="medium"
        text="123 flat, city, postcode"
      />
    </div>
  </div>
</div>
`;
